<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #0f2b49;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>911</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="SignUp">
    <widget class="QPushButton" name="registerbtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>620</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #115033; color: #fdfdfd; font-size: 23px; padding: 10px 20px; border-radius: 20px; border: 2px solid rgb(140, 140, 140);</string>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ac2323; font-size: 17px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>710</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fdfdfd; font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Уже есть аккаунт?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toLoginBtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>710</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none; color: #fdfdfd; font-size: 18px; text-decoration: underline;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="rLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>541</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arciform Sans cyr-lat</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fdfdfd; font-size: 60px; text-transform: uppercase;</string>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #17406d; border-radius: 10px; padding: 10px 30px; font-size: 20px; color: #fdfdfd</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #17406d; border-radius: 10px; padding: 10px 30px; font-size: 20px; color: #fdfdfd</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>550</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #17406d; border-radius: 10px; padding: 10px 30px; font-size: 20px; color: #fdfdfd</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Повтор пароля</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #17406d; border-radius: 10px; padding: 10px 30px; font-size: 20px; color: #fdfdfd</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Логин</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SignIn">
    <widget class="QPushButton" name="loginbtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>480</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #115033; color: #fdfdfd; font-size: 23px; padding: 10px 20px; border-radius: 20px; border: 2px solid rgb(140, 140, 140);</string>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
    <widget class="QLabel" name="lerrorLabel_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ac2323; font-size: 17px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>570</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fdfdfd; font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Ещё нет аккаунта?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toRegisterBtn">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>570</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none; color: #fdfdfd; font-size: 18px; text-decoration: underline;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>541</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arciform Sans cyr-lat</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fdfdfd; font-size: 60px; text-transform: uppercase;</string>
     </property>
     <property name="text">
      <string>вход</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #17406d; border-radius: 10px; padding: 10px 30px; font-size: 20px; color: #fdfdfd</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #17406d; border-radius: 10px; padding: 10px 30px; font-size: 20px; color: #fdfdfd</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Логин</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
